src/lib/Components/Bidding/Components/__tests__/Timer-tests.tsx

```diff
@@ -94,3 +126,53 @@ it("shows month, date, and hour adjusted for the timezone where the user is", ()
 
   expect(getTimerLabel(timer)).toEqual("Ends May 14, 1pm")
 })
+
+describe("timer transitions", () => {
```

Comment:
@yuki24 Replaced the previous tests with these which test the outcome of advancing the timer as opposed to the individual methods. In general I like to have both, but in this case it ended up being a bit redundant. 😄 

Also.... have to look at these again before merging since these are now _super_ slow.

Reply:
Ah, looks like `advanceTimersByTime` wasn't just jumping to the future, but actually fast-forwarding through execution. I moved these to only deal with single ticks (i.e. one second) and they're now normal speed.

Reply:
This looks great! Thanks for updating the tests 😀